Library/Formula/libvncserver.rb

```diff
@@ -0,0 +1,40 @@
+require "formula"
+
+class Libvncserver < Formula
+  homepage "https://libvnc.github.io/"
+  url "https://github.com/LibVNC/libvncserver/archive/LibVNCServer-0.9.10.tar.gz"
+  sha256 "ed10819a5bfbf269969f97f075939cc38273cc1b6d28bccfb0999fba489411f7"
+
+  head "https://github.com/LibVNC/libvncserver.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  depends_on "gnutls" => :recommended
+  depends_on "gtk+" => :optional
+  depends_on "libgcrypt" => :recommended
+  depends_on "jpeg" => :recommended
+  depends_on "libpng" => :recommended
+  depends_on "openssl" => :recommended
```

Comment:
Why are both OpenSSL and GnuTLS SSL/TLS toolkits recommended? Does it *use* them both? Slightly more importantly, if you choose to install with neither, does it link into the system OpenSSL in any way?